SpinCore_pp/ppg/generic.py

```diff
@@ -162,7 +166,7 @@ def generic(
                 + str(ret_data)
                 + " and we're not currently running ppgs where this makes sense"
             )
-        ret_data["indirect", indirect_idx]["nScans", nScans_idx] = data_array
+        ret_data["nScans", nScans_idx] = data_array
```

Comment:
same as above!

If you want to, you can `squeeze` at the end, but don't roll back support for the indirect axis!